test/integration_test/snapshot_restore_test.go

```diff
@@ -13,8 +13,10 @@ import (
 func testSnapshotRestore(t *testing.T) {
 	t.Run("simpleSnapshotRestoreTest", simpleSnapshotRestoreTest)
 	t.Run("groupSnapshotRestoreTest", groupSnapshotRestoreTest)
```

Comment:
Should I just add `test.Ski`p rather than commenting out  test cases that way static check will not cause issue ?

Reply:
Yes, please use `test.Skip`